.circleci/config.yml

```diff
@@ -513,14 +513,14 @@ jobs:
       - setup-rust-toolchain
       - restore_cache:
           name: Restore rubygems cache
-          key: swift-docs-gems-v13
+          key: swift-docs-gems-v15
```

Comment:
We used `-v14` in the reverted commit before. Thus skipping that and going for v15 right away.